test/e2e/smoke-test/README.md

```diff
@@ -0,0 +1,27 @@
+# Smoke Test for Packages
+
```

Comment:
May be add a little description about the scope of the smoke tests ?

Reply:
Thanks @navidshaikh for reviewing, Makes sense updating the file.